.github/workflows/ci.yml

```diff
@@ -586,16 +596,17 @@ jobs:
       - name: Untar binaries
         run: |
           mkdir -p hadoop-ozone/dist/target
-          tar xzvf target/artifacts/ozone-bin/ozone*.tar.gz -C hadoop-ozone/dist/target
-      - name: Calculate combined coverage
-        run: ./hadoop-ozone/dev-support/checks/coverage.sh
+          tar xzvf target/artifacts/ozone-bin-${{ env.TEST_JAVA_VERSION }}/ozone*.tar.gz -C hadoop-ozone/dist/target
       - name: Setup java 17
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
           java-version: 17
```

Comment:
Should we use `TEST_JAVA_VERSION` here instead of hardcoding it?

Reply:
Yes, we can.  Updated, thanks.